pyaedt/maxwell.py

```diff
@@ -1520,7 +1520,7 @@ def enable_harmonic_force(
         last_cycles_number=1,
         calculate_force="Harmonic",
     ):
-        """Set the Harmonic Force for Transient Analysis.
+        """Enable Harmonic Force calculation for Transient Analysis.
```

Comment:
```suggestion
        """Enable the harmonic force calculation for the transient analysis.
```
We are trying to avoid unnecessary capitalization because we aren't matching a UI like in the product documentation.